pkg/apis/core/validation/exposureclass.go

```diff
@@ -17,17 +17,40 @@ package validation
 import (
 	"github.com/gardener/gardener/pkg/apis/core"
 
+	apivalidation "k8s.io/apimachinery/pkg/api/validation"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 )
 
 // ValidateExposureClass validates a ExposureClass object.
 func ValidateExposureClass(exposureClass *core.ExposureClass) field.ErrorList {
-	allErrs := field.ErrorList{}
+	var allErrs = field.ErrorList{}
+
+	if len(exposureClass.Handler) == 0 {
+		allErrs = append(allErrs, field.Invalid(field.NewPath("handler"), exposureClass.Name, "exposure class handler cannot be empty"))
+	}
+
+	return append(allErrs, validateExposureClassScheduling(exposureClass.Scheduling, field.NewPath("scheduling"))...)
+}
+
+func validateExposureClassScheduling(scheduling *core.ExposureClassScheduling, fldPath *field.Path) field.ErrorList {
+	var allErrs = field.ErrorList{}
+
+	if scheduling == nil {
+		return append(allErrs, field.Required(fldPath, "exposure class need to have a scheduling configuration"))
+	}
```

Comment:
Why?
If it's not optional then the `ExposureClassScheduling` should not be a pointer in the API

Reply:
The `scheduling` section is now optional. Removed the validation.